src/metabase/pulse.clj

```diff
@@ -92,6 +92,7 @@
                     :run (fn [query info]
                            (qp/process-query-and-save-with-max-results-constraints! (assoc query :async? false) info))))]
       {:card card
+       :dashcard dashcard
```

Comment:
Including `dashcard` here so that we can later include dashboard card descriptions beneath card names